project/tests/com/google/daggerquery/executor/models/QueryTest.java

```diff
@@ -28,6 +28,8 @@
 
 public class QueryTest {
 
+  // Tests for `DEPS` query
+
```

Comment:
Would you mind moving this to be one test lower - above the testParsingDepsQuery_WithTwoStringParameters_ThrowsIllegalArgumentException test?

This test (where the query name is not provided) looks like it can be considered common testing infrastructure across all query types.